src/menus/menu.jsx

```diff
@@ -367,17 +367,17 @@ const Menu = React.createClass({
     this._setFocusIndex(focusIndex, false);
 
     if (multiple) {
-      const index = menuValue.indexOf(itemValue);
-      const newMenuValue = index === -1 ?
+      const index2 = menuValue.indexOf(itemValue);
```

Comment:
Sorry for the late response :sweat_smile: 

I think `itemIndex` is more readable.